### PR TITLE
Exercise 6 ready for review

### PR DESCRIPTION
**Pull request by aravindhan-a-cdw**
This is a question to calculate time taken by a transport to deliver parcel in destination and displaying the time at the destination timezone.

### CODE REVIEW

File: exercise_6/DeliveryTimeCalculator.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+		System.out.print("Enter distance in KM: ");
+		double distance = input.nextDouble();
+		
+		System.out.print("Enter vehicle speed in KM/Hr: ");
+		double speed = input.nextDouble();
+		
+		System.out.print("Enter number of hours left to start journey: ");
+		double hoursLeftToStart = input.nextDouble();
+		
+		System.out.println(ZoneId.getAvailableZoneIds());
```
Comment (Adheesh-Sirius) [inline exercise_6/DeliveryTimeCalculator.java]: Add exception handling while receiving input as the user might provide incorrect values. As you are getting the destination time zone, there might a chance that the user misspells the input.

File: exercise_6/DeliveryTimeCalculator.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class DeliveryTimeCalculator {
+
+	public static void main(String[] args) {
+		Scanner input = new Scanner(System.in);
```
Comment (Adheesh-Sirius) [inline exercise_6/DeliveryTimeCalculator.java]: Please add comments explaining the process. It'll help other developers understand how you are achieving the task

**Review by Adheesh-Sirius — CHANGES_REQUESTED** [review]

File: exercise_6/DeliveryTimeCalculator.java (hunk 1 above)
Comment (aravindhan-a-cdw) [reply exercise_6/DeliveryTimeCalculator.java]: Added exception handling for timezone and other inputs from user.

File: exercise_6/DeliveryTimeCalculator.java (hunk 2 above)
Comment (aravindhan-a-cdw) [reply exercise_6/DeliveryTimeCalculator.java]: Added comments and java docs to explain the process handled in the code.

**Review by Adheesh-Sirius — APPROVED** [review]

**Pull request merged**